src/Domain/Campaign.php

```diff
@@ -175,6 +175,18 @@ class Campaign extends SalesforceReadProxy
     #[ORM\Embedded(columnPrefix: 'total_fundraising_target_')]
     private Money $totalFundraisingTarget;
 
+    /**
+     * Optional BG-defined default sort override for the metacampaign grid.
```

Comment:
worth adding a bit more docs? Does higher make it go up or down the grid? What does null do?

Reply:
Done in 53d593d1e4221bbff163abd65722377c941075a0